pyiron_atomistics/vasp/interactive.py

```diff
@@ -296,6 +299,16 @@ def _run_if_new(self, debug=False):
             self._check_incar_parameter(parameter="ISYM", value=0)
         super(VaspInteractive, self)._run_if_new(debug=debug)
 
+    def convergence_check(self):
```

Comment:
Merge with the master branch which updates this function!

Reply:
Done.